pom.xml

```diff
@@ -673,6 +709,59 @@
           </archive>
         </configuration>
       </plugin>
+
+      <plugin>
+        <groupId>org.jacoco</groupId>
+        <artifactId>jacoco-maven-plugin</artifactId>
+        <version>0.8.11</version> <!-- Use the latest version available -->
```

Comment:
jacoco-maven-plugin is defined in the parent POM. If you want to update the version, there is a property for that.